agimus_controller/agimus_controller/mpc.py

```diff
@@ -0,0 +1,39 @@
+import time
```

Comment:
The class is not exposed so I will keep these measurement in python.

Reply:
It seems like the StopWatch functionalities are exposed in python, I don't since what version.

https://github.com/loco-3d/crocoddyl/blob/da92f67394c07c987458a8cb24bc0e33edd64227/bindings/python/crocoddyl/core/utils/stop-watch.cpp#L45

Reply:
Ok but we cannot add StopWatch this is specific to the solvers and C++ implementations.